runtimes/node-17.0/README.md

```diff
@@ -59,3 +86,7 @@ module.exports = (req, res) => {
     });
 }
 ```
+
+- To handle dependencies, you need to have `package.json` file. Dependencies will be automatically cached and installed, so you don't need to include `node_modules` folder in your function.
```

Comment:
Is the runtime responsible for installing the dependencies? Don't we delegate this to the builder in Appwrite? Meaning the runtime should just get everything ready for execution?

Reply:
Runtime consists of multiple scripts. One of them is `build`, one is `launch`.

Build gets package.json, and prepares folder that is ready to be executed.

Launch takes this folder (gzipped version of it) and launches it.

Also there is new `deploy.sh`, which takes these two actions and merges it. With deploy, you can point it to your code (index.js and package.json), and it will first build it (generating node_modules folder), and then run it (HTTP server).

---

Does the description make sense now, or should we update it?